c8/JWT/core.py

```diff
@@ -0,0 +1,15 @@
+from c8.request import Request
```

Comment:
@edgargarciamacrometa I don't think different folders, file and a new e2e test are required here.. 

you can just add a function in client.py called get_jwt_token with "password, email" as "mandatory" params and "username and tenant" as optional params... then I would say you can create a data object out of those params in that client function itself and call  _get_auth_token function:
https://github.com/Macrometacorp/pyC8/blob/master/c8/connection.py#L92

Note: You will need to rework the above "_get_auth_token" function adding an optional object parameter data.. which if is not None we will use that data object directly else we create a data object from self.email and self.password .. also I think we will need to tweak the response in the above function return body instead of token, tenant... and in this case, we will also need to do some changes here like add .get("tenant") and .get("jwt") to:
https://github.com/Macrometacorp/pyC8/blob/master/c8/connection.py#L59 

And for test I think you can simply add a line in of the old tests maybe test_client.py I guess..

Do let me know if there is some confusion or doubts here...